pipelinesScripts/pluginUpload.sh

```diff
@@ -86,7 +75,7 @@ buildAndUpload () {
   destPath="$JFROG_CLI_PLUGINS_RT_REGISTRY_REPO/$JFROG_CLI_PLUGIN_PLUGIN_NAME/$JFROG_CLI_PLUGIN_VERSION/$pkg/$exeName"
   echo "Uploading $exeName to $JFROG_CLI_PLUGINS_RT_REGISTRY_URL/$destPath ..."
 
-  ./jfrog rt u "./$exeName" "$destPath" --url="$JFROG_CLI_PLUGINS_RT_REGISTRY_URL" --access-token=$JFROG_CLI_PLUGINS_RT_REGISTRY_TOKEN
+  res=$(eval "./jfrog rt u ./$exeName $destPath")
```

Comment:
The result of the command doesn't seems to be used, any reason to insert into this eval?